contribute/design/1.x/upgrade/volume-pools-upgrade.md

```diff
@@ -258,6 +258,15 @@ This design proposes the following key changes:
       #this flag will be set to "no" when current != desired. 
       #after upgrading all the child objects, this will be changed to yes. 
       dependentsUpgraded:
+      #status gives the details about the reconciliation of the version
+      status:
```

Comment:
`LastUpdatedTime` to know when the status really got updated

Reply:
Done